sdk/communication/Azure.Communication.Chat/README.md

```diff
@@ -11,7 +11,7 @@ This package contains a C# SDK for Azure Communication Services for chat.
 Install the Azure Communication Chat client library for .NET with [NuGet][nuget]:
 
 ```PowerShell
-dotnet add package Azure.Communication.Chat --version 1.0.1
+dotnet add package Azure.Communication.Chat --version 1.1.0
```

Comment:
```suggestion
dotnet add package Azure.Communication.Chat 
```

We generally recommend this form, which will use the latest GA.   That avoids the need to remember to increment the README for each release.